pkg/clusters/types/gke/builder.go

```diff
@@ -26,21 +26,27 @@ type Builder struct {
 	jsonCreds         []byte
 	waitForTeardown   bool
 
-	createSubnet   bool
-	addons         clusters.Addons
-	clusterVersion *semver.Version
-	majorMinor     string
-	labels         map[string]string
+	createSubnet    bool
+	addons          clusters.Addons
+	clusterVersion  *semver.Version
+	majorMinor      string
+	nodeMachineType string
+	labels          map[string]string
 }
 
+const (
+	defaultNodeMachineType = "e2-highcpu-4"
```

Comment:
Not a blocker but I'm curious: is this something derivable from the googlecloud packages we import?

Reply:
A quick code search on Github and through https://grep.app/ doesn't show any exposed `const`s that we could use.